components/Server.coffee

```diff
@@ -16,24 +16,30 @@ exports.getComponent = ->
       error:
         datatype: 'object'
 
+  c.autoOrdering = false
   c.servers = []
+  c.context = null
 
-  c.on 'end', ->
+  c.tearDown = (done) ->
     for server in c.servers
       server._connections = 0
       server.close()
+    c.context.deactivate()
```

Comment:
Should check if context is there. It might be that you tear down a network before you ever started a server process